python/pyspark/sql/tests/connect/test_parity_dataframe.py

```diff
@@ -85,10 +81,6 @@ def test_extended_hint_types(self):
     def test_fillna(self):
         super().test_fillna()
 
-    @unittest.skip("Fails in Spark Connect, should enable.")
-    def test_freqItems(self):
```

Comment:
For this, can we change `test_freqItems` at `pyspark.sql.tests.test_dataframe` not to use `self.sc`? e.g.,:

```diff
- df = self.sc.parallelize(vals).toDF()
+ df = self.spark.createDataFrame(vals)```